storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -69,6 +69,15 @@ include::modules/lvms-unsupported-devices.adoc[leveloffset=+2]
 
 * xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-reference-file_logical-volume-manager-storage[{lvms} reference YAML file]
 
+//Using software RAID arrays
+include::modules/lvms-using-raid-arrays.adoc[leveloffset=+1]
+
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_storage_devices/managing-raid_managing-storage-devices#creating-a-software-raid-on-an-installed-system_managing-raid[Creating a software RAID on an installed system]
```

Comment:
We don't really like to link content in assemblies. We prefer to xref but I'm not sure we can do that here.

Reply:
Since the content is not available in OCP docs, links to RHEL documentation have been added.